xapian-applications/omega/index_file.cc

```diff
@@ -541,8 +542,8 @@ bool index_update_entry(const string & urlterm,DirectoryIterator &d){
             time_t doc_last_altered = binary_string_to_int(value);
             if(last_modified <= doc_last_altered && doc_last_altered <= last_altered){
                 if (verbose)
-        	    cout << "Updating entry" << endl;
-                // db.replace_document(did, doc);//UPDATE EXISTING DOCUMENT?
+        	    cout << "Updating entry without extract text" << endl;
+                db.replace_document(did, doc);//UPDATE EXISTING DOCUMENT
```

Comment:
It's not clear to me why this change is in the commit. If it's needed, then it'd be helpful for the commit message to explain why.

Reply:
It was my mistake. It is related to another issue.
I will try to make some changes.